---
layout: post
title: Summer 2021 and Summer 2022 Analyses - Getting Gene Annotations
date: '2024-09-27'
categories: [pycno]
---
I have `hisat2` and `deseq2` results from sea star coelomcyte RNAseq data from 2021 and 2022 datasets. I want to get gene annotations. See post for details.

# Annotating Count Matrices with Gene IDs
R Code: [paper-pycno-sswd-2021-2022/blob/main/code/19-get-gene-annotations.Rmd](https://github.com/grace-ac/paper-pycno-sswd-2021-2022/blob/main/code/19-get-gene-annotations.Rmd)  

Gene list IDs pulled from [augustus.hints.gtf](https://datadryad.org/stash/downloads/file_stream/2634380):     
[paper-pycno-sswd-2021-2022/data/gene_annotation.tab](https://github.com/grace-ac/paper-pycno-sswd-2021-2022/blob/main/data/gene_annotation.tab)

```
Graces-MacBook-Air-10:data graciecrandall$ head -5 gene_annotation.tab
pycn_heli.0392	AUGUSTUS	gene	1	4781	0.95	+	.	g1
pycn_heli.0392	AUGUSTUS	gene	16379	16612	0.78	-	.	g2
pycn_heli.0392	AUGUSTUS	gene	16805	17239	0.97	-	.	g3
pycn_heli.0392	AUGUSTUS	gene	17793	17996	1	-	.	g4
pycn_heli.0069	AUGUSTUS	gene	1394	26872	0.76	-	.	g5
```


This file will be `join`-ed with `DESeq2` output from analyses using the gene count matrix created in this R code: [https://github.com/grace-ac/paper-pycno-sswd-2021-2022/code/20-hisat2-genecount-matrices.Rmd](https://github.com/grace-ac/paper-pycno-sswd-2021-2022/blob/main/code/20-hisat2-genecount-matrices.Rmd)

More to come! 
